src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -195,6 +195,9 @@ void JwtAuthenticator::VerifyKey(const PubkeyCacheItem& issuer_item) {
     return;
   }
 
+  // kJwtPayloadKey is preserved for backward compatibility.
+  // Please refer to https://github.com/istio/istio/issues/4744
```

Comment:
Simply state: TODO(lei-tang): remove this backward compatibility. Tracking issue: ...

(the issue is for all tasks related to remove the old EUC policy, feel free to add note to that to remember).

Reply:
Done: https://github.com/istio/proxy/pull/1367